docs/csharp/cs-dev-kit-faq.md

```diff
@@ -103,7 +123,7 @@ This is usually because the project targets .NET Framework rather than .NET Core
 
 ![Project Not Supported in Solution Explorer](images/faq/solution-explorer-not-supported-framework.png)
 
-You will need to [update your project](https://learn.microsoft.com/dotnet/core/porting/) to an SDK-style project to use all available C# Dev Kit features.
+You can either [update your project](https://learn.microsoft.com/en-us/dotnet/core/porting/) to an SDK-style project to use all available C# Dev Kit features. Or you can use the new Prefer CSharp Extension workspace setting located in the Settings UI. This setting delegates solution and project load to the C# extension but C# Dev Kit features will be unavailable.
```

Comment:
This needs more details. Tell them how to set the setting (both options, with the toast or manually).